dio/lib/src/adapter.dart

```diff
@@ -4,6 +4,7 @@ import 'dart:typed_data';
 import 'package:meta/meta.dart';
 
 import 'adapters/io_adapter.dart'
+    if (dart.library.js_util) 'adapters/browser_adapter.dart'
```

Comment:
This should be `dart.library.js_interop` as mentioned here: https://dart.dev/interop/js-interop/package-web#conditional-imports